src/main/resources/db/migration/common/V1_401__add_user_for_automated_task.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO user_account VALUES (-5, NULL, 'AudioLinkingAutomatedTask', NULL, 'AudioLinkingAutomatedTask', '2024-01-01 00:00:00+00', '2024-01-01 00:00:00+00', NULL, 0, 0, NULL, true, true, 'AudioLinkingAutomatedTask');
```

Comment:
Do we need a new user could we not use the standard 0 one?

Reply:
the idea is that each AutomatedTask has its own user account, in this way when a modification is made to a row in the database, you know what task did it. ATM, when we go live, 2nd line support will not know which of our 30+ tasks changed the row.

Reply:
@cmagowan-hmcts suggested using a different user for each automated task to be able to distinguish the tasks who made DB changes

Reply:
Yeah @cmagowan-hmcts and I discussed this and it's a good suggestion and this is the starting point.